src/popup/router/pages/PermissionsDetails.vue

```diff
@@ -20,7 +20,7 @@
     <div class="transaction-sign-limit">
       <InputField
```

Comment:
You can also add `new-ui` prop to this input to align it with the interface design changes,

Reply:
Can be a separate PR